client/src/lib/api.js

```diff
@@ -0,0 +1,62 @@
+import axios from 'axios';
+import config from '../env/config.js';
+
+
+// This first section contains helper/template functions to keep our other functions
+// simple and DRY. Reuse them! But please don't edit them without team buy-in, because all
+// of our individual API functions rely on them.
+const api = {};
+
+const url = `https://app-hrsei-api.herokuapp.com/api/fec2/${config.CAMPUS_CODE}`;
+
+const get = (endpoint, params = {}) => {
+  return axios.get(`${url}/${endpoint}/`, {headers: {Authorization: config.AUTH}, params: params})
+    .then(results => results.data)
+    .catch(err => console.error(`Error getting ${endpoint} from server:  ${err}`));
+}
+
+const post = (endpoint, data = {}, params = {}) => {
+  return axios.post(`${url}/${endpoint}/`, data, {headers: {Authorization: config.AUTH}, params: params})
+    .then(results => results.data)
+    .catch(err => console.error(`Error getting ${endpoint} from server:  ${err}`));
+}
+
```

Comment:
Why are we returning results.data? What data will there be to return for a post?

Reply:
Sometimes post requests will return the db version of the thing you just posted.. e.g. if you add a new review and then when you send it to the database, it automatically generates a unique id and timestamp, I like to immediately "get" that back from the database to keep my client object and my db object formats in sync.

Reply:
Okay, got it. That makes sense. Good thinking!